gluten-iceberg/src/main/scala/org/apache/gluten/execution/IcebergScanTransformer.scala

```diff
@@ -56,7 +56,7 @@ case class IcebergScanTransformer(
 
   override lazy val fileFormat: ReadFileFormat = GlutenIcebergSourceUtil.getFileFormat(scan)
 
-  override def getSplitInfos: Seq[SplitInfo] = {
+  override def getSplitInfosFromPartitions(partitions: Seq[InputPartition]): Seq[SplitInfo] = {
```

Comment:
this seems a bug, can you please fire one issue to track for it?

Reply:
[5250](https://github.com/apache/incubator-gluten/issues/5250) created.